lib/features/manage_account/presentation/forward/widgets/autocomplete_contact_text_field_with_tags.dart

```diff
@@ -317,4 +325,73 @@ class _AutocompleteContactTextFieldWithTagsState extends State<AutocompleteConta
       FocusScope.of(context).unfocus();
     }
   }
+
+  void _addEmailAddressToInputFieldAction({
+    required BuildContext context,
+    required EmailAddress emailAddress
+  }) {
+    if (_isDuplicatedRecipient(emailAddress.emailAddress)) {
+      return;
+    }
+
+    final validateSameDomain = EmailUtils.isSameDomain(
+      emailAddress: emailAddress.emailAddress,
+      serverDomain: widget.serverDomain
+    );
+
+    if (!validateSameDomain) {
+      _showWarningDialogWithExternalDomain(
```

Comment:
no need await here?

Reply:
No, It already has a callback function, we just need to return after calling it to not run the next commands